src/Education/XamlGenerator/RuleHelpXamlTranslator.cs

```diff
@@ -104,23 +102,25 @@ public FlowDocument Create(IRuleInfo ruleInfo)
                             } while (xamlOutputElement.HtmlElementName != reader.Name);
 
                             break;
+
                         default:
                             var message = string.Format(Resources.XamlBuilder_UnexpectedNodeError, reader.NodeType, reader.Name, reader.Value);
                             throw new InvalidDataException(message);
                     }
                 }
 
                 // We've processed all of the html elements.
-                // Now, the stack should only contain the root FlowDocument element, plus an extra
+                // Now, the stack should only contain the root element, plus an extra
```

Comment:
one of the changes is here I think removing flowdocument makes it accurate. Assertion was already accurate.